blueprints/app/files/app/index.html

```diff
@@ -15,6 +15,10 @@
     {{content-for "head-footer"}}
   </head>
   <body>
+    <noscript>
+      JavaScript must be enabled to run this application.
```

Comment:
Because their code has to be written a certain way that guards for fastboot usage, which requires extra work, as well as the addons they use have to be fastboot compatible.